site/src/docs/about/this-is-hds/index.jsx

```diff
@@ -85,38 +85,29 @@ const DemoPage = () => {
         pageTitle="This is HDS"
         description="The Helsinki Design System is a tool to create a consistent, user-friendly and accessible digital user experience for the City of Helsinki."
       ></Seo>
-      <div className="hero-container">
-        <div className="hero-wrapper">
-          <div className="hero">
-            <div className="hero-content">
-              <div className="hero-content-shape" />
-              <Logo src={logoFi} aria-hidden className="info-page-hero-logo" />
-              <h1 className="hero-title info-page-hero-title">Helsinki Design System (HDS)</h1>
-              <p className="hero-text">
-                The Helsinki Design System focuses on usability and accessibility. It aims to improve the quality and
-                consistency of the City of Helsinki's digital services – making the user experience better for everyone.
-              </p>
-              <Button
-                variant="primary"
-                className="front-page-hero-button"
-                role="link"
-                onClick={() => {
-                  navigate('/getting-started');
-                }}
-              >
-                Visit hds.hel.fi
-              </Button>
-            </div>
-            <Koros className="hero-koros hero-koros-rotated" flipVertical rotate="45deg" />
-            <Koros className="hero-koros hero-koros-vertical" flipVertical />
-          </div>
-        </div>
-        <div
-          className="hero-bg-image"
-          style={{ backgroundImage: `url(${withPrefix('/images/about/this-is-hds/tripla.jpg')}` }}
-        />
-      </div>
-
+      <Hero
```

Comment:
In demo site hero is broken 
<img width="837" alt="image" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/75484068/173123f4-0fe0-485e-8d7c-5acb1d6602c0">